doc/whats-new.md

```diff
@@ -1,356 +1,197 @@
-Release notes for the OpenQuake Engine, version 2.2
+Release notes for the OpenQuake Engine, version 2.3
 ===================================================
 
-This release has focused on memory and performance optimizations, especially
-in the event based and scenario calculators. Moreover we have significantly
-improved our exports, by providing more information (especially for the
-event based ruptures) and/or using more efficient export formats, like the
-.npz format. As of now, the XML exports are deprecated, even if there is
-no plan to remove them at the moment.
+This release introduces several new features and improvements of the
+engine calculators. Moreover, our packaging strategy has been
+revolutionized and now we provide packages for nearly every operating
+system - including old ones like Ubuntu 12.04, previously abandoned - by
+using the exact same versions of the libraries on every supported
```

Comment:
"... packages for most common operating systems - including previously abandoned systems such as Ubuntu 12.04 - by ..."